### ISSUE TITLE
[android 5] The insecure lock icon is white on the toolbar

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/focus-android#6022]
## Steps to reproduce
1. On an Android 5 devices, install Focus, and set the Light theme.
2. Open an insecure page.
3. Observe the insecure lock icon from the toolbar.

### Expected behavior
The insecure lock icon is visible.

### Actual behavior
The insecure lock icon is white, and not visible.

### Device information
* Android device: Samsung Galaxy Tab a6 (Android 5.1.1), Xiaomi mi4i (Android 5.0.2)
* Focus version: Beta 96.0.0-beta.1
* Not reproducible on Focus Nightly 95.0 (353421248 GV 96.0a1-20211204095400)

![white icon](https://user-images.githubusercontent.com/40694158/145380902-123c5c50-21a7-455b-9a7f-d0764b952eb9.jpg)

**Comment by ionutbedregeanu** [comment]
@lobontiumira Still reproducing? I tried on Pixel 5 Android 5.1 and I didn't succeed

**Comment by lobontiumira** [comment]
Yes, I am still able to reproduce this issue on Xiaomi mi4i (Android 5.0.2), on Focus Nightly 97.0 (build 360070846 GV 97.0a1-20220106155649):

![Screenshot_2022-01-11-15-49-50-658_org mozilla focus nightly](https://user-images.githubusercontent.com/40694158/148955185-c2b1a8a7-40b9-42e1-b6a4-927b17718d7d.png)

**Comment by ionutbedregeanu** [comment]
https://github.com/mozilla-mobile/android-components/issues/11537 should fix the bug

**Comment by Mugurell** [comment]
Looked into this more with Ionut and Mira and saw that this issue happens only after closing a tab, not with the first tab opened from a cold start and this seems like an intermittent issue, with cases in which this will work even in this scenario.

This seems to all depend on the fact that when a tab is closed we are resetting (clearing) the list of indicator and colors used for them and then seemingly there is an edgecase in which these are not properly updated when loading a new page.

**Comment by ionutbedregeanu** [comment]
To validate the fix we need to wait https://github.com/mozilla-mobile/android-components/pull/11546 to be merged

**Comment by delia-pop** [comment]
This is fixed on Focus Nightly 98.0 (build 360280510 GV 98.0a1-20220127094620) with Huawei MediaPad M2 (Android 5.1.1). 
Note that is still present in Beta 97.0.0-beta.6.
